library/src/main/java/com/prolificinteractive/materialcalendarview/MaterialCalendarView.java

```diff
@@ -1145,6 +1148,7 @@ public void writeToParcel(@NonNull Parcel out, int flags) {
             out.writeInt(dynamicHeightEnabled ? 1 : 0);
             out.writeInt(calendarMode == CalendarMode.WEEKS ? 1 : 0);
             out.writeParcelable(currentMonth, 0);
+            out.writeByte((byte) (saveCurrentPosition ? 1 : 0));
```

Comment:
Is a boolean writable in a more direct manner than mapping it to a byte of 0 or 1?